README.md

```diff
@@ -226,6 +226,8 @@ Methods defined by `DayViewState` class.
 | animateToPage | int page | Animate to page index defined by `page`. |
 | jumpToDate | DateTime date | Jumps to the page that has a calendar for month defined by `date` |
 | animateToDate | DateTime date | Animate to the page that has a calendar for month defined by `date` |
+| animateToEvent | CalendarEventData event | Animates to the page where given `event` is and then scrolls to make that `event` visible on screen. |
```

Comment:
There is a small grammar mistake here. 

"Animates to the page where a given event is and then scrolls to make that event visible on the screen.


Jumps to the page where a given event is and then scrolls to make that event visible on the screen."

Reply:
Please check the same for the below strings as well.